### PR TITLE
To-do list

### PR DESCRIPTION
**Pull request by ShaliniNayan**
- Project with webpack that is based on the webpack exercise.
- Function to iterate over the tasks array and populate an HTML list item element for each task.
- All source files (index.html, index.js and style.css) are located in the src directory.
- Create an index.js file and set an array of some simple to-do tasks (array of objects). Each task object contain three 
   keys: 
         i) description [string].
        ii) completed [bool].
       iii) index: [number].

### CODE REVIEW

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @ShaliniNayan 👋,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

# Status: Invalid Code Review Request



You have submitted a project, that uses the wrong README file. Kindly find the [updated readme file here.](https://github.com/microverseinc/readme-template).

**Note that only the FAQ section is optional.**
**The Live Demo section is only optional if the project does not require deployment.**

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by Gambit142 — COMMENTED** [review]
Hi @ShaliniNayan 👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- Kindly remove the I've Demo Section from the README.md file. When I click on the Live Demo section, it displays your README.
- Please include `npm install` as the way to install all dependencies and `npm start` as the way to start the dev server

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by DammyShittu — COMMENTED** [review]
Hi @ShaliniNayan,

While you made a great effort in this project, I cannot proceed to review your code.

## Invalid Code Review Request

### Kindly push the change requested by the previous reviewer

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: dist/index.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        <img src="/src/icon/verticle-menu.png" alt="menu" />
+      </label>
+      <br />
+      <input type="checkbox" id="myCheck" value="todo" />
+      <label for="todo-list">
+        complete To Do list project
+        <img src="/src/icon/return-icon.png" alt="return" /> </label
+      ><br /><br />
+      <button type="button" class="clear-btn" id="clear-btn">
+        Clear all completed
```
Comment (frank1738) [inline dist/index.html:34]: - [ ] Kindly note that in this milestone you are supposed to set an array of some simple-to-do tasks (array of objects). Each task object should contain three key 
  - description [string].
  - completed [bool].
  - index: [number].
- [ ] And then Write a function to iterate over the tasks array and populate an HTML list item element for each task. Currently, your array is empty

**Review by frank1738 — CHANGES_REQUESTED** [review]
Hi @ShaliniNayan , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

✅ Correct use of Github workflow
✅ No linter error


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

N/A
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by addod19 — CHANGES_REQUESTED** [review]
### Hi @ShaliniNayan , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- [ ] Nice job with the awesome work you have done so far :rocket: 


 ## Required Changes ♻️
_Check the comments under the review._
- [x] Kindly fix the errors I'm getting running or starting the app as you can see in the screenshots below
Screenshot from 2023-05-03 19-33-24
Screenshot from 2023-05-03 19-33-06


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ShaliniNayan** [comment]
image

**Comment by ShaliniNayan** [comment]
image

**Comment by ShaliniNayan** [comment]
There is no error showing on my computer.

**Review by V-Blaze — APPROVED** [review]
Hi @ShaliniNayan  , 

# Approved 🟢 🟢 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
giphy

## Highlights
- Webpack setup correctly✔
- Todo items displayed in the browser on page load✔
- CSS is loaded using the `Webpack Style/CSS Loader.` :+1: :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**